NetworkingDemo/DemoRepository.swift

```diff
@@ -30,7 +30,7 @@ internal class DemoRepository: AbstractRepository {
     }
     
     public func noAnswerEntities(userID: Int) -> SignalProducer<Void, RepositoryError> {
-        let path = DemoRepository.ReadPrefixPath + "/\(userID)" + DemoRepository.ReadSuffixPath
+        let path = DemoRepository.ReadPrefixPath + "\(userID)" + DemoRepository.ReadSuffixPath
```

Comment:
@danielaRiesgo how would you refactor that code using `appendingPathComponent`? I cannot find a nice way.